package Dash::Exceptions::CallbackException;

use Moo;

with 'Throwable';

1;

